plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcWriteConfig.java

```diff
@@ -22,8 +22,10 @@
 public class JdbcWriteConfig
 {
     static final int MAX_ALLOWED_WRITE_BATCH_SIZE = 1_000_000;
+    static final int DEFAULT_WRITE_PARALELLISM = 8;
 
     private int writeBatchSize = 1000;
+    private int writeParallelism = DEFAULT_WRITE_PARALELLISM;
```

Comment:
can we default to `max(4, num_workers)` or `max(8, num_workers)` instead? For larger clusters this would be a regression.